README.md

```diff
@@ -1 +1,5 @@
-# microverse-redo-trials
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please give a more descriptive and professional README file. I give you this link for a good README template: [template](https://github.com/microverseinc/readme-template)

Reply:
Great job here but as mentioned in the summary it lacks an explanation. Be more specific about what you're looking for in the README file. What information should be included? For example, you could say something like:

"The README file should be more descriptive and professional. It should include a brief overview of the project, as well as installation instructions, usage examples, and a list of contributors. You can use this link for an example of a good README template: [template](https://github.com/microverseinc/readme-template)"